src/api/Recording.api.js

```diff
@@ -58,6 +59,25 @@ function tracks(recordingId) {
   });
 }
 
+function replaceTrackTag(tag, recordingId, trackId) {
+  const body = querystring.stringify({
+    what: tag.what,
+    confidence: tag.confidence,
+    automatic: "false",
+    data: JSON.stringify("")
```

Comment:
can this not just be left out?

Reply:
yep, just have to set a default in api server